runtime/nls/vmut/j9vmutil.nls

```diff
@@ -191,9 +191,9 @@ J9NLS_VMUTIL_RECEIVERCLASS_NOT_SAME_OR_SUBTYPE_OF_CURRENTCLASS.system_action=
 J9NLS_VMUTIL_RECEIVERCLASS_NOT_SAME_OR_SUBTYPE_OF_CURRENTCLASS.user_response=
 # END NON-TRANSLATABLE
 
-J9NLS_VMUTIL_OPENJ9_EXTENDED_HCR_DISABLED_WARNING=Warning: Extended HCR is disabled, enable with -XX:+EnableExtendedHCR.
+J9NLS_VMUTIL_OPENJ9_EXTENDED_HCR_DISABLED_WARNING=Warning: Extended HCR is disabled, enable with -XX:+EnableExtendedHCR. In the future, the extended HCR capability will be removed from existing and future releases.
```

Comment:
Not sure "will be removed from existing and future releases" is clear. It can't be removed from pre-existing releases.
How about "the extended HCR capability, and the option to enabled it, will be completely removed."

Reply:
Updated my last comment.